components/ide/jetbrains/gateway-plugin/src/main/kotlin/io/gitpod/jetbrains/gateway/GitpodWorkspacesView.kt

```diff
@@ -124,9 +124,21 @@ class GitpodWorkspacesView(
                         override fun actionPerformed(e: AnActionEvent) {
                             BrowserUtil.browse("https://${settings.gitpodHost}")
                         }
+                    }, object : DumbAwareAction("See Usage", "See Usage", AllIcons.Actions.DynamicUsages) {
+                        override fun actionPerformed(e: AnActionEvent) {
+                            BrowserUtil.browse("https://${settings.gitpodHost}/usage")
+                        }
                     }, object : DumbAwareAction("Documentation", "Documentation", AllIcons.Toolwindows.Documentation) {
                         override fun actionPerformed(e: AnActionEvent) {
-                            BrowserUtil.browse("https://www.gitpod.io/docs/ides-and-editors/jetbrains-gateway")
+                            BrowserUtil.browse("https://www.gitpod.io/docs/integrations/jetbrains-gateway")
+                        }
+                    }, object : DumbAwareAction("Send Feedback", "Send Feedback", AllIcons.Actions.IntentionBulb) {
+                        override fun actionPerformed(e: AnActionEvent) {
+                            BrowserUtil.browse("https://github.com/gitpod-io/gitpod/issues/6576")
+                        }
+                    }, object : DumbAwareAction("Help", "Help", AllIcons.Actions.Help) {
```

Comment:
How about removing the verbs `Open`, `See` and `Send` to leave them closer to the Menu from the Dashboard?
So the options would be:
- Dashboard
- Usage
- Documentation
- Feedback
- Help
- Log Out

<img width="226" alt="image" src="https://user-images.githubusercontent.com/418083/211499344-b8e00d8e-196f-4f49-88b8-280262bb0112.png">

Reply:
yeah, this makes sense. Will fix it :)

Thanks for the review 🙏🏽